client/web/src/search/results/SearchAlert.tsx

```diff
@@ -26,7 +26,16 @@ export const SearchAlert: React.FunctionComponent<React.PropsWithChildren<Search
     <Alert className="my-2" data-testid="alert-container" variant="info">
         <H3>{alert.title}</H3>
 
-        {alert.description && <Markdown className="mb-3" dangerousInnerHTML={renderMarkdown(alert.description)} />}
+        {alert.description && (
+            <Markdown
+                className="mb-3"
+                dangerousInnerHTML={renderMarkdown(alert.description, {
+                    // Disable autolinks to rev specs are not rendered as email links
```

Comment:
I don't think I'd understand this comment without the context of this PR. Maybe this version?
```suggestion
                    // Disable autolinks to prevent revision specifications from being rendered as email links
```

Reply:
Yes, typo here 😅